Functions.Templates/Bindings/bindings.json

```diff
@@ -1910,8 +1914,12 @@
           "name": "disable",
           "type": "NAND",
           "values": [
-            { "value": "idToken" },
```

Comment:
I checked. This is the same as the code in azure git hub repo

---
In reply to: [133859502](https://github.com/Azure/azure-webjobs-sdk-templates/pull/539#discussion_r133859502) [](ancestors = 133859502)